manifests/crane/params.pp

```diff
@@ -6,6 +6,6 @@
   $debug = false
   $port = 5000
 
-  $data_dir = '/var/lib/crane/metadata'
+  $data_dir = '/var/lib/pulp/published/docker/v2/app'
```

Comment:
I think you'd add a line here: https://github.com/Katello/puppet-katello/blob/master/manifests/init.pp#L124

$crane_data_dir => '/var/lib/pulp/published/docker/v2/app'